shared/teams/confirm-modals/confirm-kick-out.tsx

```diff
@@ -28,15 +28,15 @@ const ConfirmKickOut = (props: Props) => {
     setWaiting(true)
     const teamMemberToRemoves = members.map(member => {
       return {
-        type: 0, // TODO how do I get RPCTypes.TeamMemberToRemoveType.assertion to typecheck?
         assertion: {
           assertion: member,
           removeFromSubtree: subteamsToo,
         },
+        type: 0, // TODO how do I get RPCTypes.TeamMemberToRemoveType.assertion to typecheck?
```

Comment:
I think make it `members.map<RPCTypes.teamsTeamRemoveMemberParam>` above. my type name is likely wrong, but it should exist.